runtime/libpgmath/lib/common/dispatch.c

```diff
@@ -1264,7 +1281,8 @@ __math_dispatch_init()
   if (__sync_bool_compare_and_swap(&__math_dispatch_in_prog, false, true)) {
     if (__mth_i_debug == 0x100) {
       fputs("calling __math_dispatch()\n", stderr);
-      sleep(1);
+      struct timespec tsp = { 0, 250000000 };
```

Comment:
What is the reason to use nanosleep over sleep?

Reply:
sleep is an approximation. it may or may not sleep for the number of seconds in the argument.

nanosleep is precise.